modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+- To enable storing complex WebDAV properties, a https://github.com/owncloud/core/pull/37314[migration step] adds a new column (`propertytype`) to the `oc_properties` and `oc_dav_properties` tables. The impact on upgrade duration depends on the number of rows in `oc_properties` and `oc_dav_properties`.
+- To facilitate the transition to the new licensing mechanism (xref:release_notes.html#changes-in-delivery[see below]) a https://github.com/owncloud/core/pull/37512[migration step] will disable the `enterprise_key` app during the upgrade. This step does not have an impact on upgrade duration.
+
+=== PHP 7.1 Support Discontinued
+
+As xref:release_notes.html#php-71-deprecation-note[announced], in the previous minor release of ownCloud Server, from version 10.5 onward, ownCloud Server **no longer supports PHP 7.1**. 
+If you're running on PHP 7.1 or below, it is necessary to upgrade PHP **prior** to conducting the upgrade to Server 10.5. 
+See the xref:installation/system_requirements.adoc[system requirements] for more information.
+
+NOTE: If you're using the official Docker containers or the Univention appliance, this has been taken care of already.
+
+=== Official PHP 7.4 Support
+
+ownCloud Server 10.5 officially supports PHP 7.4. 
+The Server Core and all apps maintained by ownCloud have received a full QA cycle and are proven to work reliably with PHP 7.4.
+If you are still running a PHP version < 7.2, you must upgrade PHP before upgrading ownCloud Server as lower versions are not supported anymore.
+
+Summarizing, ownCloud Server 10.5 supports the PHP versions **7.2, 7.3 and 7.4**.
+
+TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud Documentation] for more information.
+
+NOTE: The official ownCloud Docker containers have been updated to Ubuntu 20.04 and are using PHP 7.4. 
+
+=== File Locking in the Web Interface
+
+ownCloud Server 10.5 comes with great enhancements for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out). 
+
+The feature builds on the xref:release_notes.html#webdav-locks[WebDAV Locks backend] which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
```

Comment:
`xref:release_notes.html#webdav-locks` --> `xref:release_notes.adoc#webdav-locks`
You have to referene to a source document (pls check the path...)
(`xref:module_name:<path>/file.adoc#section[Printed Name]`)